projects/step-frontend/src/lib/os-plugins/modules/script-editor/components/script-editor/script-editor.component.scss

```diff
@@ -1,15 +1,27 @@
 :host {
   display: block;
   padding: 1.6rem;
-  width: 100%;
+  max-width: 100%;
 }
 
 mat-card-header {
   display: flex;
   justify-content: space-between;
   align-items: center;
+
+  .color-green {
+    color: green;
```

Comment:
@lukasz-lepa Please, import file with variables and apply colors from it.
As for example you can look at `incident-status.component.scss`

Reply:
File imported